src/Upgrade.tsx

```diff
@@ -173,15 +214,10 @@ const style = StyleSheet.create({
     flex: 1,
     marginTop: topInset,
   },
-  contentContainer: {
-    flexGrow: 1,
-    paddingTop: Sizing.layout.x20,
-    paddingBottom: Sizing.layout.x60,
-    paddingHorizontal: Sizing.layout.x20,
-  },
   innerContainer: {
-    flexGrow: 1,
     justifyContent: "space-between",
+    paddingTop: Sizing.layout.x20,
```

Comment:
```suggestion
    paddingTop: Sizing.x20,
```
And for any other cases where we use `layout`.

Reply:
dont currently have this in our styles. was planning on taking a whole pass at all of em in a separate pr.